src/services/firebase-client/index.ts

```diff
@@ -0,0 +1,3 @@
+import * as rtdb from './rtdb'
+
+export const db = rtdb
```

Comment:
I'm unfamiliar with realtime db from firebase
not sure if this is necessary

Reply:
this is not a must. I did this only to make `firebase.db` be abstract (inversion control)

Reply:
If so, I think you should encapsulate persistence layer per data model.

```
user_store/service/users/whatever {
    create(user)
    get(id)
    list(filter?)
    ...
}

redemption/service/whatever {
    create(user)
    get(id)
    list(filter?)
    redempt?
    ...
}
```